### DISCUSSION TITLE
丧一会 | /posts/sang-yi-hui/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#581] [Announcements]
## 丧一会

https://yuanfan.rbind.io/posts/sang-yi-hui/

**Comment by residualsun1** [comment]
且不说这种与多个自我对话的写作形式很有趣独特，我觉得如果身体里还有很多个自己在“打架”，这也是一件幸运的事情，感觉自我仍然是那么地丰富和具有生命力。

**Reply by earfanfan** [reply]
讲真，每次有陌生的朋友夸我一顿，我都会偷着乐很久很久。哈哈哈，我决定未来要专门抽点时间写一篇博客，名为“我的朋友把我夸上天系列”，专门收录像你这样夸赞我的文字，并且持续不断地更新。俺继续偷着乐去也……

**Comment by yihui** [comment]
不知我是否该对这个链接名表示微微抗议。
